aries-site/src/examples/components/table/TableFixedHeaderExample.js

```diff
@@ -0,0 +1,241 @@
+  },
+  {
+    id: '44:07:0b:ba:1e:eb',
+    macAddress: '44:07:0b:ba:1e:eb',
+    ipAddress: '192.168.0.18',
+    hostname: 'wh7-dureal50',
+    connectionType: 'Wireless',
+    category: 'Network',
+    status: 'Active',
+    deviceName: 'Google-Home-Mini',
+    networkBand: { value: 2.4, unit: 'GHz' },
+    linkRate: { value: 72, unit: 'Mbps' },
+  },
+  {
+    id: '00:f6:20:60:15:d0',
+    macAddress: '00:f6:20:60:15:d0',
+    ipAddress: '192.168.0.17',
+    hostname: 'wh7-dureal50',
+    connectionType: 'Wireless',
+    category: 'Network',
+    status: 'Active',
+    deviceName: 'Amazon-Alexa',
+    networkBand: { value: 2.4, unit: 'GHz' },
+    linkRate: { value: 72, unit: 'Mbps' },
+  },
+  {
+    id: '30:fd:38:88:49:f4',
+    macAddress: '30:fd:38:88:49:f4',
+    ipAddress: '192.168.0.16',
+    hostname: 'wh7-dureal50',
+    connectionType: 'Wireless',
+    category: 'Network',
+    status: 'Active',
+    deviceName: 'Google-Home-Mini',
+    networkBand: { value: 2.4, unit: 'GHz' },
+    linkRate: { value: 292, unit: 'Mbps' },
+  },
+  {
+    id: '7c:b0:c2:5c:94:a5',
+    macAddress: '7c:b0:c2:5c:94:a5',
+    ipAddress: '192.168.0.26',
+    hostname: 'wh7-dureal50',
+    connectionType: 'Wireless',
+    category: 'Network',
+    status: 'Active',
+    deviceName: 'N/A',
+    networkBand: { value: 2.4, unit: 'GHz' },
+    linkRate: { value: 6, unit: 'Mbps' },
+  },
+  {
+    id: 'E0:5F:45:96:F4:DD',
+    macAddress: 'E0:5F:45:96:F4:DD',
+    ipAddress: '192.168.0.25',
+    hostname: 'wh7-dureal50',
+    connectionType: 'Wireless',
+    category: 'Network',
+    status: 'Active',
+    deviceName: 'N/A',
+    networkBand: { value: 2.4, unit: 'GHz' },
+    linkRate: { value: 1, unit: 'Mbps' },
+  },
+];
+
+const columns = [
+  {
+    primary: true,
+    property: 'macAddress',
+    header: 'MAC Address',
+    render: datum => datum.macAddress,
+    pin: true,
+  },
+  {
+    property: 'ipAddress',
+    header: 'IP V4',
+    render: datum => <Text truncate>{datum.ipAddress}</Text>,
+    size: 'xsmall',
+  },
+  {
+    property: 'hostname',
+    header: 'Hostname',
+    render: datum => <Text truncate>{datum.hostname}</Text>,
+    size: 'xsmall',
+  },
+  {
+    property: 'connectionType',
+    header: 'Connection Type',
+  },
+  {
+    property: 'category',
+    header: 'Category',
+  },
+  {
+    property: 'deviceName',
+    header: 'Device Name',
+    render: datum => <Text truncate>{datum.deviceName}</Text>,
+    size: 'xsmall',
+  },
+  {
+    property: 'status',
+    header: 'Status',
+  },
+  {
+    property: 'networkBand.value',
+    header: 'Network Band (GHz)',
+    render: datum => datum.networkBand.value,
+    align: 'end',
+  },
+  {
+    property: 'linkRate.value',
+    header: 'Link Rate (Mbps)',
+    render: datum => datum.linkRate.value,
+    align: 'end',
+  },
+];
+
+export const TableFixedHeaderExample = () => {
+  return (
+    <Box alignSelf="center">
+      <Heading level={3} margin="none">
```

Comment:
This example on Firefox is squishing the columns together/not allowing horizontal scroll:
<img width="855" alt="Screen Shot 2020-09-14 at 3 44 57 PM" src="https://user-images.githubusercontent.com/12522275/93145627-ac5a7b80-f6a1-11ea-9104-9552dcf64e62.png">

Reply:
Addressed.

Similar to: https://github.com/hpe-design/design-system/pull/1089#discussion_r488971526